docs/core/compatibility/breaking-changes.md

```diff
@@ -12,7 +12,7 @@ The following version selector provides a filtered list of breaking changes for
 You can also view individual issues that detail the breaking changes introduced in .NET Core 3.0 in the following GitHub repositories:
 
 - For .NET Core, the [dotnet/docs](https://github.com/dotnet/docs/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking-change) repo.
```

Comment:
should we say something like this instead since all ASP.NET issues were ported here too?

```suggestion
- For .NET Core and ASP.NET Core, the [dotnet/docs](https://github.com/dotnet/docs/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking-change) repository.
```